### DISCUSSION TITLE
Recommended IDE

### DISCUSSION THREAD

**Discussion by Zageron** [airsdk/Adobe-Runtime-Support#935] [Q&A] [answered]
Hello,

Our team currently uses Visual Studio Code with [asconfigc](https://github.com/BowlerHatLLC/asconfigc).
Is this a recommended toolchain still? Or should we be considering switching to Moonshine or IntelliJ? (We switched from Flash Develop to VSC.)

Thanks!

**Comment by marchbold** [comment, marked as answer]
I would say VS code with the as extension is still a very valid IDE! (I assume you are using the [extension](https://github.com/BowlerHatLLC/vscode-as3mxml) not just the utility?)

We use it in several of our projects but mainly we use IntelliJ

**Reply by Zageron** [reply]
Yes, we do use the extension. :)
Thanks for confirming that it's still a good choice.
